pkgs/development/python-modules/buildout-nix/default.nix

```diff
@@ -1,11 +1,11 @@
 { fetchurl, stdenv, buildPythonPackage }:
 
 buildPythonPackage {
-  name = "zc.buildout-nix-2.4.0";
+  name = "zc.buildout-nix-2.4.3";
 
   src = fetchurl {
-    url = "https://pypi.python.org/packages/source/z/zc.buildout/zc.buildout-2.4.0.tar.gz";
-    md5 = "b8323b1ad285544de0c3dc14ee76ddd3";
+    url = "https://pypi.python.org/packages/source/z/zc.buildout/zc.buildout-2.4.3.tar.gz";
+    md5 = "32dcb3de0673193b78d670c29551ef6c";
```

Comment:
when we're updating packages, make sure you convert md5 (deprecated) to sha256. Thanks!